package.json

```diff
@@ -39,6 +39,7 @@
     "@angular/compiler": "^8.0.3",
     "@angular/compiler-cli": "^8.0.3",
     "@angular/core": "^8.0.3",
+    "@angular/commons": "^8.0.3",
```

Comment:
It looks like you have missed to commit `package-lock.json` (that is why the build is failing)

Reply:
Should be ok now with the proper dependency